testlib/src/anker_context.rs

```diff
@@ -118,12 +125,11 @@ impl Context {
         let b_sol_mint = solido_context.create_mint(b_sol_mint_authority).await;
         let payer = solido_context.context.payer.pubkey();
 
-        let token_pool_context = initialize_token_pool(&mut solido_context).await;
+        let token_pool_context = setup_token_pool(&mut solido_context).await;
 
         let rewards_owner = solido_context.deterministic_keypair.new_keypair();
-        let ust_rewards_account = solido_context
-            .create_spl_token_account(token_pool_context.ust_mint_address, rewards_owner.pubkey())
-            .await;
+        // TODO: Replace with a valid Terra address.
```

Comment:
Hmm, so I looked briefly into this, it looks like Terra addresses are variable-length? Found this: <https://docs.rs/cosmwasm-terra/0.7.2/cosmwasm_terra/types/struct.CanonicalAddr.html>. and also this: https://docs.cosmos.network/master/basics/accounts.html#keys-accounts-addresses-and-signatures.

So it looks like they support multiple signature schemes, and addresses are byte arrays? We will have to dive a bit deeper later.

Reply:
👀  We will, can you write an issue about it?